src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,397 @@
+
+### Step 1: Provider onboarding
+
+In this step, the provider connects their Flex account with the payment
+gateway. This is where they provide the bank details that eventually
+receive money from the customers. In addition, in this step, they
+provide the necessary information and documents for the identity
+verification and _Know Your Customer_
+[(KYC)](https://en.wikipedia.org/wiki/Know_your_customer) requirements.
+[These requirements vary](https://stripe.com/docs/connect/required-verification-information)
+depending on the user’s country of residence.
+
+### Step 2: Customer checkout
+
+Customer checkout happens when the customer initiates a transaction. At
+this stage, they also provide the payment information, such as their
+credit card number. Also, the payment will be made at this point. The
+payment gateway will preauthorize the money, i.e. reserve the money on
+the customer's credit card.
+
+### Step 3: Provider acceptance
+
+After the customer has checked out, the provider has the ability to
+either accept or reject the request. If the request is accepted, the
+payment will be captured, and the reserved money will be transferred
+from the customer's credit card to the payment gateway.
+
+This is a step that you can combine with the customer checkout. The flow
+where the “provider accept” happens instantly after customer checkout is
+called [instant booking](#instant-booking) flow.
+
+### Step 4: Customer refund
+
+Typically, the marketplace payment flow contains a delayed payment
+period. This is the time between when the money is captured from the
+customer's credit card and when it is transferred to the provider's bank
+account. The payout in marketplaces usually happens after the provider
+has successfully provided the agreed service.
+
+Customer refund usually happens during this delayed payment period.
+There are many reasons why a refund may be necessary. For example, the
+provider or customer may not be able to make it, or the provided service
+was not what was agreed.
+
+### Step 5: Provider payout
+
+If everything in the transaction went right and the customer received
+the agreed service, the money from the payment gateway will be
+eventually paid out to the provider.
+
+## Stripe default integration
+
+In the default Flex implementation, the process described above is
+implemented using Stripe. The integration uses
+[Stripe Custom Connect accounts](https://stripe.com/docs/connect/custom-accounts)
+as the provider’s payout account. The customer can checkout using a
+payment card, and they can also save their payment method for future
+use. The integration uses
+[Stripe destination charges](https://stripe.com/docs/connect/destination-charges)
+to collect the payment to the platform’s account first. Once the
+transaction is successfully over, the provider’s share of the payment is
+paid through the provider’s Custom Connect account to the bank account
+they provided upon onboarding.
```

Comment:
IIRC the amount is first transferred to the provider's Connect account (using destination charges?). From there possible commission is paid to the operator's Stripe platform account using Stripe's application fee. Pay out then pays the remaining part to the provider's bank account from the Connect account.

Reply:
Clarified!